docs/uperf.md

```diff
@@ -35,6 +35,34 @@ Once done creating/editing the resource file, you can run it by:
 # kubectl apply -f <path_to_file> # if created a new cr file
 ```
 
+Note: If you'd like to try to experiment with storing results in a pv and have followed
+instructions to deploy operator with attached pvc and would like to send results to ES.
+you can instead define a custom resource like
+```yaml
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  es:
+    server: <es_host>
+    port: <es_port>
+    index: <es_index>
+  user: rht_perf_ci
```

Comment:
Add context here, the `user:` here is the one executing the test. This will make finding results easier in ES.

Reply:
Agreed will update the document.